### ISSUE TITLE
Build failure "This extension does not yet support the current version of JupyterLab"

### ISSUE THREAD

**Issue by ocramz** [DataHaskell/data-glue#3]
```
 ---> 3f30e27b3c82
Step 6/29 : RUN jupyter labextension install ihaskell_jupyterlab
 ---> Running in 60c211a49f30
> /usr/bin/npm pack ihaskell_jupyterlab
ihaskell_jupyterlab-0.0.6.tgz

Errored, use --debug for full output:
ValueError: This extension does not yet support the current version of JupyterLab.


Conflicting Dependencies:
JupyterLab              Extension        Package
>=0.18.2 <0.19.0        >=0.17.2 <0.18.0 @jupyterlab/apputils
>=0.18.2 <0.19.0        >=0.17.1 <0.18.0 @jupyterlab/notebook
>=0.18.2 <0.19.0        >=0.17.1 <0.18.0 @jupyterlab/application
```

**Comment by dleprovost** [comment]
Thanks for the notice.
The Dockerfile needs to be refreshed a bit, I'm on it.

**Comment by dleprovost** [comment]
For now, we fix `0.33` as the version of jupyterlab to use.
A patch for make `ihaskell_jupyterlab` running on jupyterlab `0.34` is on its way.

Bytheway, we should now always fix the version, to avoid future occurrences of this.
